.travis.yml

```diff
@@ -1,39 +1,43 @@
 language: python
+addons:
+  firefox: '61.0.2'
```

Comment:
Why this version? I'm thinking we use the `latest-esr` alias here to stay on ESR.

Reply:
I just took the same version as it was [before](https://github.com/RedHatQE/widgetastic.core/pull/118/files#diff-354f30a63fb0907d4ad57269548329e3L25), but of course `latest-esr` makes more sense.